test/Microsoft.NET.Sdk.Functions.Generator.Tests/FunctionJsonConverterTests.cs

```diff
@@ -24,6 +26,24 @@ public class FunctionsClass
             [FunctionName("MyHttpTrigger")]
             public static void Run1([HttpTrigger] HttpRequestMessage request) { }
 
+            [FunctionName("HttpTriggerWriteToQueue1")]
+            [return: Queue("myqueue-items-a", Connection = "MyStorageConnStra")]
+            public static string HttpTriggerWriteToQueue1([HttpTrigger] HttpRequestMessage request) => "foo";
+
+            [FunctionName("HttpTriggerWriteToQueue2")]
+            public static void HttpTriggerWriteToQueue2([HttpTrigger] HttpRequestMessage request,
+                [Queue("myqueue-items-b", Connection = "MyStorageConnStrb")] out string msg)
+            {
+                msg = "foo";
+            }
+
+            [FunctionName("HttpTriggerWriteToQueue3")]
```

Comment:
How about HttpTriggerMultipleQueueOutputs, verifying IAsyncCollector as well as the ablility for a function to have multiple outputs (say a Blob output as well?)

Reply:
Added as suggested.